### PR TITLE
Check if the configured URL is valid.

### PR DESCRIPTION
**Pull request by stepardo**
Informs the user if the URL is not valid. This fixes issue #4.

### CODE REVIEW

**Comment by stepardo** [comment]
Maybe somebody else can add a french translation.

**Comment by nicosomb** [comment]
```
cfg_msg_text=Merci de configurer l'URL de votre instance de wallabag.
cfg_invalid_msg_text=L'URL configurée est invalide. Merci de la corriger.
```

File: index.js [hunk 1]
```diff
@@ -35,6 +36,15 @@ function wallabagBagIt(url) {
         var am = require("sdk/preferences/utils");
         am.open(self);
         return;
+    } else if (!urls.isValidURI(wallabagUrl)) {
```
Comment (Aldarone) [inline index.js]: There is a return in the previous condition, why not simplify by dropping the else and making it a separate if ?

**Comment by stepardo** [comment]
Hi,

thanks for the translation. I add the cfg_invalid_msg_text. Should I 
update the cfg_msg_text as well? Since I don't speak french, I have no 
idea which tranlation is better ;-)


On 01.02.2016 21:56, Nicolas Lœuillet wrote:
> |cfg_msg_text=Merci de configurer l'URL de votre instance de wallabag.
> cfg_invalid_msg_text=L'URL configurée est invalide. Merci de la corriger. |

Best,
Steffen

**Comment by nicosomb** [comment]
You're right, cfg_msg_text is good.

File: index.js (hunk 1 above)
Comment (stepardo) [inline index.js]: Changed.

**Comment by Aldarone** [comment]
+1

**Pull request merged**